examples/pytorch/MNIST/README.md

```diff
@@ -0,0 +1,3 @@
+## MNIST Handwritten Digit Recognition examples
+
+This folder contains two subdirectories: `example1`, which demonstrates MNIST autolog example with Early stopping and model checkpoint, `example2`, demonstrates MNIST autolog example where metrics are logged every n iterations.
```

Comment:
I'm ok with merging this as is in this PR, but in a follow-up could we just combine these two examples into a single one? It seems we could show logging every N iterations via a single commented-out line of code in ``example1`` (e.g. have a commented-out line like):

```
# Replace the existing mlflow.pytorch.autolog() call with the one below to log training metrics
# every 10 epochs, instead of the default of logging on every epoch
# mlflow.pytorch.autolog(every_n_epoch=10)
mlflow.pytorch.autolog()
```